locales/en-US/app.l20n

```diff
@@ -95,3 +95,7 @@
 <tourDoneButton "Done">
 <userCountContainer "There are <span class=\"bold\">{{installation_count}}</span> people trying {{title}} right now!">
 <highlightPrivacy "Your privacy">
+
+<noScriptHeading "Uh oh...">
```

Comment:
FWIW, L20N as we currently use it requires JS. So, these strings won't get touched in the noscript case.

There *is* a [python-l20n](https://github.com/l20n/python-l20n) that seems current. Probably worth a follow-up issue to research how we might be able to use that for server-generated pages like this.